solutions/only_ints.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for verifying integer inputs.
+
+Module contents:
+    - only_ints: A function to check if two parameters are both integers.
+
+Created on 12 29 2024
```

Comment:
Maybe put data in some format like yyyy.mm.dd or yyyy/mm/dd?

Reply:
Agree , I will do it